src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.cpp

```diff
@@ -0,0 +1,658 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.hpp"
+
+#include <algorithm>
+
+#include "Utilities/GenerateInstantiations.hpp"
+
+namespace GeneralizedHarmonic::BoundaryConditions::Bjorhus {
+namespace detail {
+template <size_t VolumeDim, typename DataType>
+void add_gauge_sommerfeld_terms_to_dt_v_minus(
+    const gsl::not_null<tnsr::aa<DataType, VolumeDim, Frame::Inertial>*>
+        bc_dt_v_minus,
+    const Scalar<DataType>& gamma2,
+    const tnsr::I<DataType, VolumeDim, Frame::Inertial>& inertial_coords,
+    const tnsr::a<DataType, VolumeDim, Frame::Inertial>& incoming_null_one_form,
+    const tnsr::a<DataType, VolumeDim, Frame::Inertial>& outgoing_null_one_form,
+    const tnsr::A<DataType, VolumeDim, Frame::Inertial>& incoming_null_vector,
+    const tnsr::A<DataType, VolumeDim, Frame::Inertial>& outgoing_null_vector,
+    const tnsr::Ab<DataType, VolumeDim, Frame::Inertial>& projection_Ab,
+    const tnsr::aa<DataType, VolumeDim, Frame::Inertial>&
+        char_projected_rhs_dt_v_psi) noexcept {
+  // gauge_bc_coeff below is hard-coded here to its default value in SpEC
+  constexpr double gauge_bc_coeff = 1.;
+
+  DataType inertial_radius_or_scalar_factor(get_size(get<0>(inertial_coords)),
+                                            0.);
+  for (size_t i = 0; i < VolumeDim; ++i) {
+    inertial_radius_or_scalar_factor += square(inertial_coords.get(i));
+  }
+  inertial_radius_or_scalar_factor =
+      get(gamma2) -
+      gauge_bc_coeff * (1. / sqrt(inertial_radius_or_scalar_factor));
```

Comment:
can you write this as `gauge_bc_coeff / sqrt(inertial_radius_or_scalar_factor)`? I'm not sure the compiler is smart enough to optimize out the unnecessary multiplication by 1.0

Reply:
done

Reply:
done